packages/amazonq/src/extensionCommon.ts

```diff
@@ -113,9 +112,6 @@ export async function activateAmazonQCommon(context: vscode.ExtensionContext, is
     // reload webviews
     await vscode.commands.executeCommand('workbench.action.webview.reloadWebviewAction')
 
-    // enable auto suggestions on activation
-    await CodeSuggestionsState.instance.setSuggestionsEnabled(true)
```

Comment:
don't we have cache key-value to use here?
```
const value = get(autoTriggerEnabledKey, extContext)
await CodeSuggestionsState.instance.setSuggestionsEnabled(value)
```

Reply:
There was a logic to reset the AutoTrigger enabled status, was it removed after some refactoring?

Reply:
Based on my testing, without your proposed code, the extension memorizes the previous auto trigger settings without an issue.

Reply:
When the statusBarMenu is rendered, it calls `getAmazonQCodeWhispererNodes`, which calls `CodeSuggestionsState.instance.isSuggestionsEnabled()`, which reads from ` get(autoTriggerEnabledKey, this.#context)`. 

Therefore, below extra code is not needed.
```
const value = get(autoTriggerEnabledKey, extContext)
await CodeSuggestionsState.instance.setSuggestionsEnabled(value)

```

Reply:
understood, as long as we are memorizing and using users' previous config then it's fine. we can close this conversation.